.github/workflows/kind.yml

```diff
@@ -96,7 +96,7 @@ jobs:
       run: |
         make fmt
         git diff --exit-code
-        make lint
+        echo "INFO: Linting disabled."
```

Comment:
Nice

Reply:
Yeah, I realized a bit late there should be a message rather than just a removal. At least I got it right for the last batch!